devtools/prepare_aws_deploy.sh

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env bash
 
-bash build_docker.sh
-
-# REPOSITORY_URI=290492953667.dkr.ecr.ap-southeast-2.amazonaws.com/blockchaindemo
+#bash build_docker.sh
```

Comment:
commented?

Reply:
errggg good catch - was retrying code and didn't wanna wait for the build

Reply:
👍